12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,25 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+	ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+	ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[aeio] [u]
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[!aeio] [!u]
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+	ls /src *.c
   6. All files with extension ".c" under any folder
   Answer:
-
+	ls *.c
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
-
+	rm *.png -r
```

Comment:
the -r flag is redundant here unless you want to delete directories that end with ".png".. note that ```mydir.png``` is actually a valid directory name... :)  

also it's a good practice to put your flags before the 
arguments ```rm -r *.png``` somewhat because of style and somewhat because  *.png glob is expended by the shell before the command is executed and if ever gonna write/debug shell scripts, it's gonna be hard to the -r see...